ui/redux/actions/collections.js

```diff
@@ -377,7 +381,7 @@ export const doFetchItemsInCollection =
       collectionItems = await promisedCollectionItemsFetch;
     }
 
-    if (!collectionItems || collectionItems.length === 0) {
+    if (!collectionItems || collectionItems?.length === undefined) {
```

Comment:
Forgot to mention about that.   
Changing it to just ` if (!collectionItems)` gave some lint error in https://github.com/OdyseeTeam/odysee-frontend/blob/3879843f603081dde8407a9bd1670108dd69706c/ui/redux/actions/collections.js#L413

![2024-02-22_12-15](https://github.com/OdyseeTeam/odysee-frontend/assets/34790748/e75a34e3-2fa3-450c-9609-6f2b338e69c6)

(I didn't understood the error, but that worked to avoid it)

Reply:
The original code did worked here too, but changing it to just ` if (!collectionItems) {`  gives lint error. 

Don't want to return early for 0 items, due to public list with 0 items wouldn't get "items-succesfully-resolved" status, and couldn't be edited.

Letting it continue to the end seemed to work fine. And for cases where there are issues with fetching the items, the `collectionItems` would be `null`, in which case we want to return early.  

And if there aren't issues, but there just aren't any claims(all deleted), `collectionItems` would be an empty list, and will be considered to have items successfully resolved.